src/components/FileUploader.tsx

```diff
@@ -40,13 +44,45 @@ const FileUploader: React.FC<FileUploaderProps> = ({
           className="absolute inset-0 w-full h-full opacity-0 cursor-pointer"
         />
         <div className="flex flex-nowrap text-left rounded-lg z-50">
-          <div className="max-w-max flex items-center text-white font-medium text-sm leading-5 px-4 py-2.5 rounded-l-lg border border-r-0 border-gray-300 bg-gray-800 cursor-pointer">
-            Choose file
-          </div>
-          <div className="bg-white flex-1 text-gray-900 p-3 text-sm leading-4 rounded-r-lg border border-l-0 border-gray-300">
-            {document ? "File uploaded" : " Click to upload more"}
-          </div>
+          {document ? (
+            <>
+              <div className="max-w-max flex items-center text-white font-medium text-sm leading-5 px-4 py-2.5 rounded-l-lg border border-r-0 border-green-500 bg-green-400">
+                Choose file
+              </div>
+              <div className="bg-white flex-1 text-gray-900 p-3 text-sm leading-4 rounded-r-lg border border-l-0 border-green-500 bg-green-50 text-green-700">
+                Click to upload more
+              </div>
+            </>
+          ) : (
+            <>
+              <div className="max-w-max flex items-center text-white font-medium text-sm leading-5 px-4 py-2.5 rounded-l-lg border border-r-0 border-gray-300 bg-gray-800 cursor-pointer">
+                Choose file
+              </div>
+              <div className="bg-white flex-1 text-gray-900 p-3 text-sm leading-4 rounded-r-lg border border-l-0 border-gray-300">
+                {document ? "File uploaded" : " Click to upload more"}
+              </div>
```

Comment:
Since you have already checked the condition for documents, why is there a need for an additional conditional check for the same condition?